storybook/.storybook/addons.js

```diff
@@ -1,5 +1,5 @@
-import '@storybook/addon-actions/register';
 import '@storybook/addon-knobs/register';
+import '@storybook/addon-actions/register';
```

Comment:
Any reason for changing order?

Reply:
Yes, so apparently in Storybook the order of imports reflects the order tabs will be shown on screen. In this case, it is more important to have knobs showing in the first place.

Reply:
Ok! 🤷‍♂